lib/pharos/scripts/configure-etcd.sh

```diff
@@ -82,7 +82,12 @@ fi
 
 echo "Waiting etcd to launch on port 2380..."
 
-while ! nc -z ${PEER_IP} 2380; do
+etcd_healthy() {
+  response=$(curl -s --cacert /etc/pharos/pki/ca.pem --cert /etc/pharos/pki/etcd/client.pem --key /etc/pharos/pki/etcd/client-key.pem https://${PEER_IP}:2379/health)
+  [ "${response}" = '{"health": "true"}' ]
```

Comment:
Something like `jq -r .health <<<"$response"` would be less whitespace-sensitive, but I don't think `jq` is actually installed by default.